ArduPlane/quadplane.h

```diff
@@ -632,6 +632,9 @@ class QuadPlane
     // ignored unless OPTION_DELAY_ARMING or OPTION_TILT_DISARMED is set
     bool delay_arming;
 
+    // should we force use of fixed wing controller for attitude upset recovery?
+    bool force_fw_control_recovery;
```

Comment:
I would like to see this in `QTUN`, it could be a new bit in the assistance bitmask.

https://github.com/ArduPilot/ardupilot/blob/9fb1e2b9f7b746716397aa1d298943c201d9dd8a/ArduPlane/quadplane.cpp#L3593-L3600

Reply:
added, thanks